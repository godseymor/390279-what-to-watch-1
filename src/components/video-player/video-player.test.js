import React from "react";
import renderer from "react-test-renderer";
import VideoPlayer from "./video-player.jsx";

const mocks = {poster: `poster`, preview: `preview`};
describe(`VideoPlayer:`, () => {
  it(`Ð¡orrectly rendered after reload`, () => {
    const tree = renderer.create(<VideoPlayer {...mocks} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
