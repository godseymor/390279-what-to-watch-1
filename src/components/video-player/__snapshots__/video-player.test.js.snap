// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer: Ð¡orrectly rendered after reload 1`] = `
<video
  height="175"
  muted={true}
  poster="poster"
  src="preview"
  width="280"
/>
`;
